go.mod

```diff
@@ -22,6 +23,12 @@ require (
 	gopkg.in/yaml.v2 v2.4.0
 )
 
-replace github.com/jfrog/jfrog-client-go => github.com/jfrog/jfrog-client-go v1.5.3-0.20211108143613-1665c40274c2
+replace github.com/jfrog/jfrog-client-go => github.com/jfrog/jfrog-client-go v1.6.1-0.20211116141412-7dc86e653b1a
 
-//replace github.com/jfrog/jfrog-cli-core/v2 => github.com/jfrog/jfrog-cli-core/v2 v2.4.2
+replace github.com/jfrog/jfrog-cli-core/v2 => github.com/asafgabai/jfrog-cli-core/v2 v2.0.0-20211117072556-740f530cb48f
```

Comment:
Reminder for myself to update the go.mod after the [jfrog-cli-core PR](https://github.com/jfrog/jfrog-cli-core/pull/250) is merged.